samples/user-interface/windowmanager/src/main/res/xml/main_split_config.xml

```diff
@@ -6,7 +6,7 @@
   you may not use this file except in compliance with the License.
   You may obtain a copy of the License at
 
-       https://www.apache.org/licenses/LICENSE-2.0
+       http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
why?

Reply:
On the browser it redirects to https, but http is the recommended text to add in the header: https://www.apache.org/licenses/LICENSE-2.0#apply